networks/suzuka/indexer/bin/load_metadata.rs

```diff
@@ -18,8 +18,10 @@ async fn main() -> Result<(), anyhow::Error> {
 
 	// Replace the postgres connection definition in the metadata file
 	// with the provided in the env var INDEXER_V2_POSTGRES_URL
-	let postgres_host = std::env::var("POSTGRES_DB_HOST").unwrap_or("postgres".to_string());
-	let postgres_url = format!("postgres://postgres:password@{postgres_host}:5432/postgres");
+	//	let postgres_host = std::env::var("POSTGRES_DB_HOST").unwrap_or("postgres".to_string());
+	//	let postgres_url = format!("postgres://postgres:password@{postgres_host}:5432/postgres");
```

Comment:
I would just remove the commented out lines. Or if they are informational add into a README.md.

Reply:
I've forgotten to remove it. This one should be removed.